.docker/jats-ingester/airflow.cfg

```diff
@@ -15,7 +15,7 @@ article_assets_url = https://%(destination_bucket_name)s.s3.amazonaws.com
 service_name = scholarly-articles
 
 # url to send the resulting xml
-service_url = http://unstable--api-gateway.libero.pub:8081/scholarly-articles
+service_url = https://unstable--api-gateway.libero.pub/scholarly-articles
```

Comment:
goes through HTTPS, no need for ports now, all checks out

Reply:
theoretically this could go directly to the `api-gateway` container if we wanted to use HTTP and skip some hops?

Reply:
so just `http://api-gateway:8081`?